Networking/HTTP/HTTPLogic.cc

```diff
@@ -109,9 +109,15 @@ namespace litecore { namespace net {
             rq << "CONNECT " << string(slice(_address.hostname)) << ":" << _address.port;
         } else {
             rq << MethodName(_method) << " ";
-            if (_proxy && _proxy->type == ProxyType::HTTP)
-                rq << string(_address.url());
-            else
+            if (_proxy && _proxy->type == ProxyType::HTTP) {
```

Comment:
What happens in the case of `ProxyType::HTTPS`?

Reply:
I don't know that it has HTTPS in the enum as well. The original code doesn't handle that so that seems to be a bug here. It should handle the same way.

Reply:
Will fix this.

Reply:
My guess is that it is meant to connect to a proxy that is serving SSL?  So the handshake needs to happen?  In which case it is not relevant here and we probably don't support it?

Reply:
Yes, the handshake should happen with the proxy the same way when no-proxy is used, but regarding the HTTP message, it should be the same when using HTTP and HTTPS. 

Note: I have never verified with HTTPS proxy so whether there could be a handshake issue or not, it's a separate issue.